src/DynamoManipulation/Gizmo.cs

```diff
@@ -0,0 +1,174 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Media.Media3D;
+using Autodesk.DesignScript.Interfaces;
+using Dynamo.Visualization;
+using Dynamo.Wpf.ViewModels.Watch3D;
+using ProtoCore.AST;
+using Point = Autodesk.DesignScript.Geometry.Point;
+using Vector = Autodesk.DesignScript.Geometry.Vector;
+
+namespace Dynamo.Manipulation
+{
+    /// <summary>
+    /// Interface to define visuals of a manipulator, called Gizmo.
+    /// </summary>
+    public interface IGizmo : IDisposable
```

Comment:
moved the `IGizmo` interface definition from `TranslationGizmo.cs` to this new file